src/data/metadata.csv

```diff
@@ -18418,6 +18418,14 @@ SH,SH,Saint Helena,,,,,,,,-1
 SI,SI,Slovenia,,,,,,,,-1
 SJ,SJ,Svalbard and Jan Mayen,,,,,,,,-1
 SK,SK,Slovakia,,,,,,,,-1
+SK_BC,SK,Slovakia,BC,Banskobystrický,,,,,,
+SK_BL,SK,Slovakia,BL,Bratislavský,,,,,,
+SK_KI,SK,Slovakia,KI,Košický,,,,,,
+SK_NI,SK,Slovakia,NI,Nitriansky,,,,,,
+SK_PV,SK,Slovakia,PV,Prešovský,,,,,,
+SK_TA,SK,Slovakia,TA,Trnavský,,,,,,
+SK_TC,SK,Slovakia,TC,Trenčiansky,,,,,,
+SK_ZI,SK,Slovakia,ZI,Žilinský,,,,,,
```

Comment:
It looks like these are all included in `knowledge_graph.csv` already, correct?

Reply:
Yep.